docs/hyn/5.2/Logging.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Logging
+icon: fal fa-book
+---
+
+Especially in production you might want to split the default laravel log by tenants.
+Tenancy shipps with a custom tenant aware logger which you can register in the logger config.
+
+If a valid hostname is detected it will create a `logs` folder within the tenant directory ie.:
+`app/tenancy/tenants/xxxxxxxxxxxxxxxxxxxx/logs/{log_level}_xxxx-xx-xx.log`
```

Comment:
Could you clarify where `app/....` is rooted within the project? The next example starts with `/storage...`, which clearly identifies it as a path starting from the project root.  `app/...` does not provide such context.

Reply:
Yes right, should be /storage instead of /app.